storageconnector/pgconnector.go

```diff
@@ -0,0 +1,68 @@
+package storageconnector
+
+import (
+	"database/sql"
+	"fmt"
+	_ "github.com/lib/pq"
+	"regexp"
```

Comment:
It's a good practice to sort the imports, first should come std library sorted alphabetically and then after the empty line some external libraries like `pq`